### ISSUE TITLE
Try out docker ros images

### ISSUE THREAD

**Issue by BearCatMicro** [BearCatMicro/DockerizeROS#2]
After pulling the docker ros images, we gonna start a container using that images.

Try the command below to run the container

`docker run --name rosbuild ros:indigo-ros-base-xenial`

After running the command above, a container named `rosbuild` will be started
It uses the docker image `ros:indigo-ros-base-xenial` as it base. 

Right after the start the container will go into `exited` state.
You can see this using command 
`docker container ps -a`

---------------------------
You can think of a container lives in run time and an image live in compile (build) time. 
Container have states: `created`, `restarting`, `running`, `pause`, `exited`, `dead`

In the next issues we gonna see why our container was immediately went into `exited` state, and also how to keep the container running.

**Comment by embengoibo** [comment]
Note that: Container rosbuild has been existed. It should be another name

**Comment by BearCatMicro** [comment]
I guess that I tried the same name yesterday. You could chose any random name.
Or you could stop and remove the old container by using these commands

`docker container stop rosbuild`
`docker container rm rosbuild`

**Comment by embengoibo** [comment]
- I tested with ros:indigo-ros-base instead of rosbuild ros:indigo-ros-base-xenial (because in the first issue I did comment that there is no image rosbuild ros:indigo-ros-base-xenial).
- The behavior as you described here. It immediately went to Exited within 3 seconds.
- Note that: To list the container, we should use docker container ls -a (instead docker container ps -a, maybe you have a typo?)
